apps/frontend/src/library/__tests__/ResearchOutput.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import nock from 'nock';
+import { authTestUtils } from '@asap-hub/react-components';
+import { ResearchOutputResponse } from '@asap-hub/model';
+
+import ResearchOutput from '../ResearchOutput';
+import { API_BASE_URL } from '../../config';
+
+const researchOutput: ResearchOutputResponse = {
+  id: 'f9712278-b3f2-4895-8e4b-d5ebdb7f49b7',
+  created: '2020-09-24T16:18:23Z',
+  url: '',
+  doi: '',
+  type: 'proposal',
+  title: 'Proposal title.',
+  text: 'Actual proposal?',
+  publishDate: '2020-02-02T12:00:00.000Z',
+};
+
+const renderComponent = async (id: string) => {
+  const result = render(
+    <authTestUtils.Auth0Provider>
+      <authTestUtils.WhenReady>
+        <authTestUtils.LoggedIn user={undefined}>
+          <MemoryRouter initialEntries={[`/${id}/`]}>
+            <Route path="/:id" component={ResearchOutput} />
+          </MemoryRouter>
+        </authTestUtils.LoggedIn>
+      </authTestUtils.WhenReady>
+    </authTestUtils.Auth0Provider>,
+  );
+  await waitFor(() =>
+    expect(result.queryByText(/auth0/i)).not.toBeInTheDocument(),
+  );
+  await waitFor(() =>
+    expect(result.queryByText(/loading/i)).not.toBeInTheDocument(),
+  );
+  return result;
+};
+afterEach(() => {
+  nock.cleanAll();
+});
+it('renders the proposal', async () => {
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/research-outputs/42')
+    .once()
+    .reply(200, researchOutput);
```

Comment:
Can I ask you to re-state the `title` here (`{...researchOutput, title: 'asdf'}`) please :)